packages/cli-lib/api/marketplaceValidation.js

```diff
@@ -0,0 +1,45 @@
+const http = require('../http');
+
+const VALIDATION_API_BASE = 'quality-engine/v1/validation';
+
+/**
+ * @async
+ * @param {number} accountId
+ * @returns {Promise}
+ */
+async function requestValidation(accountId, body = {}) {
```

Comment:
Do these need to be async if there is no usage of await within them?

Reply:
Let me look that up, good question.

Reply:
you can remove async here!